metis-harvesting/src/main/java/eu/europeana/metis/harvesting/http/HttpHarvesterImpl.java

```diff
@@ -177,7 +181,7 @@ public void forEachFiltered(ReportingIteration<FullRecord> action, Predicate<Pat
     }
   }
 
-  private static class PathIterator extends AbstractHttpHarvestIterator<Path> {
+  public static class PathIterator extends AbstractHttpHarvestIterator<Path> {
```

Comment:
I suppose this now is a reusable class outside of the containing. Should we convert it to a top level class with javadoc?